src/styles/blocks/calendar.scss

```
@@ -0,0 +1,42 @@
+.calendar {
+  width: $day-size * 7 + $day-gap * 6;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+
+  &__day {
+    width: $day-size;
+    height: $day-size;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid #000;
+    background-color: #eee;
+    transition-duration: 500ms;
+
```

by default transition using for `all`. Better to set what you need
```suggestion
    transition: background-color 0.5s, transform 0.5s;

```

---

Thank you, done. Transition properties specified.